Loop/Models/ServiceAuthentication/ShareService.swift

```diff
@@ -36,14 +38,23 @@ class ShareService: ServiceAuthentication {
                 value: url?.absoluteString,
                 options: [
                     (title: NSLocalizedString("US", comment: "U.S. share server option title"),
-                     value: DexcomShareURL.absoluteString)
+                     value: DexcomShareURL.absoluteString),
```

Comment:
DexcomShareURL should be replaced everywhere with the new string constants from Mark’s lib